Python/Product/VSCode/AnalysisVsc/PathsWatcher.cs

```diff
@@ -40,19 +43,26 @@ internal sealed class PathsWatcher : IDisposable {
             var list = new List<FileSystemWatcher>();
             var reduced = ReduceToCommonRoots(paths);
             foreach (var p in reduced) {
-                var fsw = new FileSystemWatcher(p);
-                fsw.IncludeSubdirectories = true;
-                fsw.EnableRaisingEvents = true;
+                if(!Directory.Exists(p)) {
```

Comment:
Let's just catch `IOException` here

Reply:
Ctor actually throws `ArgumentException`

Reply:
Ok. Do we care about `PathTooLongException`?

Reply:
Oh, I see